how-to-prosper-at-meetups/readme.md

```diff
@@ -0,0 +1,40 @@
+> Goal: Explain what makes Meetups fun, how you can easily socialize and how you can tell your story.
+
+# How to prosper at meetups
+
+Listen to people. Try to find something to talk about in which you are genuinely interested (find common ground). They will inadvertently ask what you are doing.
+
+## General tips:
+
+### Do:
+- Be engaged by what is happening (the talk, the people around you, the location). These are all things you automatically share with all the other people that are there.
+- Be actively involved during the presentation. Ask questions and/or answer questions others have asked if you can provide an interesting perspective/ add knowledge. People will come to you after the presentations since they know what you're interested in/can provide.
+- People won't mind you joining them in a conversation, not even if they are in a deep conversation. Simply ask: "May I join you?".
+- Realize that it's perfectly fine to listen to a group and only minutes later say something (once your understand the topic and can add value to the conversation).
+- Join the conversation of the people you think are interesting; for example, those who asked interesting questions or whose Meetup profile has some interesting information.
+- Be honest and humble! Nobody expects you to be an expert at everything. Your experience in other fields, or your view as a novice in the field, can be very interesting to other people.
+- If you don't understand the topic after a few minutes, ask them to explain the foundation of the conversation. Conversations are very fluid and it's usually difficult to understand the conversation if you don't get the foundation/goal. It likely helps them have a more focussed conversation as well.
+- If you don't find the topic interesting, try to learn why the other party finds it interesting. It might be that you can expand your horizons! Don't talk too long to annoying people, though ;)
+- Learn
+
+### Don't:
+- Don't make the reason you're there about selling something; you are mostly interested in the event and the story of the other party.
+- Don’t look at your mobile phone. People won’t approach you and you will put out a negative vibe.
+- Don't have 2 on 1 conversations. This can feel like an interrogation, and speaking as a team to more people in an evening is generally more worthwhile for everybody involved.
+- Don't feel obliged to keep having a conversation with people you can't find common ground with.
```

Comment:
Just one last suggestion: Since you formatted the "Do" list with "Do" at the top as the beginning of every list item, maybe it's good to format the "Don't" list in the same way?

Ex:
Don't:
- Make the reason you're there about...